### PR TITLE
Adding in rel and classes for quick eval activities.

### PR DESCRIPTION
**Pull request by CarolBusman**

### CODE REVIEW

**Comment by CarolBusman** [comment]
Here's the route for the evaluation-status: http://search.dev.d2l/source/xref/Lms/le/activities/D2L.LE.Activities.Extensibility/Api/Hypermedia/ActivitiesController.Patterns.cs#36

File: index.js [hunk 1]
```diff
@@ -171,10 +172,13 @@ export const Rels = {
 export const Classes = {
 	activities: {
 		assigned: 'assigned',
+		assignmentActivity: 'assignment-activity',
```
Comment (ryantmer) [inline index.js]: Out of curiosity, how are these used? I would have expected just `assignment` to be sufficient, but I'm probably not thinking of the right use case!

Comment (CarolBusman) [reply index.js]: They are used to tell the difference between the activity type (assignment vs quiz vs discussion/topic). I believe the `-activity` on the classes is to be explicit out-of-context.

Comment (CodeBaboon) [reply index.js]: Should `out-of-context` or something just be a different class then? So like: `classes: ['assignment', 'activity','out-of-context']` or whatnot?

**Pull request merged**